models/kustomization.go

```diff
@@ -0,0 +1,62 @@
+package models
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+)
+
+// Komponent represents a Kustomize Component. A Component is a collection of
+// resources that can be included in a Kustomization file.
+type Komponent struct {
+	Resource  `yaml:",inline"`
+	Resources []string `yaml:",omitempty"`
+}
+
+// Kustomization represents a kustomization file.
+type Kustomization struct {
+	Komponent  `yaml:",inline"`
```

Comment:
IMO it's better to just list `Resource` / `Resources` fields here again, right now it seems like a `Komponent` _belongs_ in a `Kustomization`